#OpenXT:Since eth0 is added as a bridge slave for brbridged, dhcp is disabled by default on eth0, this patch enables it.
--- a/src/nm-ip4-config.c
+++ b/src/nm-ip4-config.c
@@ -540,8 +540,8 @@ nm_ip4_config_create_setting (const NMIP
 
 	if (!config) {
 		g_object_set (s_ip4,
-		              NM_SETTING_IP_CONFIG_METHOD, NM_SETTING_IP4_CONFIG_METHOD_DISABLED,
-		              NULL);
+		              NM_SETTING_IP_CONFIG_METHOD, NM_SETTING_IP4_CONFIG_METHOD_AUTO,
+		              NULL);
 		return NM_SETTING (s_ip4);
 	}
 
@@ -586,7 +586,7 @@ nm_ip4_config_create_setting (const NMIP
 
 	/* Use 'disabled' if the method wasn't previously set */
 	if (!method)
-		method = NM_SETTING_IP4_CONFIG_METHOD_DISABLED;
+		method = NM_SETTING_IP4_CONFIG_METHOD_AUTO;
 
 	g_object_set (s_ip4,
 	              NM_SETTING_IP_CONFIG_METHOD, method,
